packages/playground/worker/__tests__/es/es-worker.spec.ts

```diff
@@ -100,3 +104,12 @@ test('emit chunk', async () => {
     '"A string/es/"'
   )
 })
+
+test('import.meta.glob in worker', async () => {
+  expect(await page.textContent('.importMetaGlob-worker')).toMatch('["')
+})
+
+// FIXME after buildEsbuildPlugin renderChunk export default a() break token
+// test('import.meta.globEager in worker', async () => {
+//   expect(await page.textContent('.importMetaGlobEager-worker')).toMatch('["')
+// })
```

Comment:
I don't know how to solve it here, if you can help I would be very grateful

Reply:
I found that `export default` is always added after this plugin

Reply:
It looks like `--format=esm` + `module` variable is doing something.

[esbuild repl with `--minify --format=esm` including `module` variable](https://hyrious.me/esbuild-repl/?version=0.14.36&mode=transform&input=+var+module+%3D+%27A+string%27%3B%0A%0Avar+__glob_0_0+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++%27default%27%3A+module%0A%7D%29%3B%0A%0Aconst+msg1+%3D+%27module1%27%3B%0A%0Avar+__glob_0_1+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg1%3A+msg1%0A%7D%29%3B%0A%0Aconst+msg2+%3D+%27module2%27%3B%0A%0Avar+__glob_0_2+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg2%3A+msg2%2C%0A++msg1%3A+msg1%0A%7D%29%3B%0A%0Aconst+msg3+%3D+%27module3%27%3B%0A%0Avar+__glob_0_3+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg3%3A+msg3%0A%7D%29%3B%0A%0Aconst+msg+%3D+%27pong%27%3B%0Aconst+mode+%3D+%22production%22%3B%0A%0Avar+__glob_0_4+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg%3A+msg%2C%0A++mode%3A+mode%0A%7D%29%3B%0A%0Aconst+modules+%3D+%7B+%22.%2Fmodules%2Fmodule.js%22%3A+__glob_0_0%2C+%22.%2Fmodules%2Fmodule1.js%22%3A+__glob_0_1%2C+%0A%22.%2Fmodules%2Fmodule2.js%22%3A+__glob_0_2%2C+%22.%2Fmodules%2Fmodule3.js%22%3A+__glob_0_3%2C+%22.%2Fmodules%2FworkerImport.js%22%3A+__glob_0_4%2C%7D%3B%0A%0Aself.onmessage+%3D+function+%28e%29+%7B%0A++self.postMessage%28Object.keys%28modules%29%29%3B%0A%7D%3B&options=--minify+--format%3Desm)
[esbuild repl with `--minify` including `module` variable](https://hyrious.me/esbuild-repl/?version=0.14.36&mode=transform&input=+var+module+%3D+%27A+string%27%3B%0A%0Avar+__glob_0_0+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++%27default%27%3A+module%0A%7D%29%3B%0A%0Aconst+msg1+%3D+%27module1%27%3B%0A%0Avar+__glob_0_1+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg1%3A+msg1%0A%7D%29%3B%0A%0Aconst+msg2+%3D+%27module2%27%3B%0A%0Avar+__glob_0_2+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg2%3A+msg2%2C%0A++msg1%3A+msg1%0A%7D%29%3B%0A%0Aconst+msg3+%3D+%27module3%27%3B%0A%0Avar+__glob_0_3+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg3%3A+msg3%0A%7D%29%3B%0A%0Aconst+msg+%3D+%27pong%27%3B%0Aconst+mode+%3D+%22production%22%3B%0A%0Avar+__glob_0_4+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg%3A+msg%2C%0A++mode%3A+mode%0A%7D%29%3B%0A%0Aconst+modules+%3D+%7B+%22.%2Fmodules%2Fmodule.js%22%3A+__glob_0_0%2C+%22.%2Fmodules%2Fmodule1.js%22%3A+__glob_0_1%2C+%0A%22.%2Fmodules%2Fmodule2.js%22%3A+__glob_0_2%2C+%22.%2Fmodules%2Fmodule3.js%22%3A+__glob_0_3%2C+%22.%2Fmodules%2FworkerImport.js%22%3A+__glob_0_4%2C%7D%3B%0A%0Aself.onmessage+%3D+function+%28e%29+%7B%0A++self.postMessage%28Object.keys%28modules%29%29%3B%0A%7D%3B&options=--minify)
[esbuild repl with `--minify --format=esm` not including `module` variable](https://hyrious.me/esbuild-repl/?version=0.14.36&mode=transform&input=var+module11+%3D+%27A+string%27%3B%0A%0Avar+__glob_0_0+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++%27default%27%3A+module11%0A%7D%29%3B%0A%0Aconst+msg1+%3D+%27module1%27%3B%0A%0Avar+__glob_0_1+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg1%3A+msg1%0A%7D%29%3B%0A%0Aconst+msg2+%3D+%27module2%27%3B%0A%0Avar+__glob_0_2+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg2%3A+msg2%2C%0A++msg1%3A+msg1%0A%7D%29%3B%0A%0Aconst+msg3+%3D+%27module3%27%3B%0A%0Avar+__glob_0_3+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg3%3A+msg3%0A%7D%29%3B%0A%0Aconst+msg+%3D+%27pong%27%3B%0Aconst+mode+%3D+%22production%22%3B%0A%0Avar+__glob_0_4+%3D+%2F*%23__PURE__*%2FObject.freeze%28%7B%0A++__proto__%3A+null%2C%0A++msg%3A+msg%2C%0A++mode%3A+mode%0A%7D%29%3B%0A%0Aconst+modules+%3D+%7B+%22.%2Fmodules%2Fmodule.js%22%3A+__glob_0_0%2C+%22.%2Fmodules%2Fmodule1.js%22%3A+__glob_0_1%2C+%0A%22.%2Fmodules%2Fmodule2.js%22%3A+__glob_0_2%2C+%22.%2Fmodules%2Fmodule3.js%22%3A+__glob_0_3%2C+%22.%2Fmodules%2FworkerImport.js%22%3A+__glob_0_4%2C%7D%3B%0A%0Aself.onmessage+%3D+function+%28e%29+%7B%0A++self.postMessage%28Object.keys%28modules%29%29%3B%0A%7D%3B&options=--minify+--format%3Desm)